src/js/structs/Pod.js

```diff
@@ -138,6 +138,10 @@ module.exports = class Pod extends Service {
     return this._spec;
   }
 
+  getStatusString() {
+    return (this.get('status') || '').toUpperCase();
```

Comment:
There are cases we know, such as this:
https://github.com/mesosphere/marathon/blob/feature/pods/docs/docs/rest-api/public/api/v2/types/podStatus.raml#L100

But also cases were we don't know what it could be, like this:
https://github.com/mesosphere/marathon/blob/feature/pods/docs/docs/rest-api/public/api/v2/types/podStatus.raml#L36